src/LibraryManager/Providers/Unpkg/UnpkgProvider.cs

```diff
@@ -39,7 +39,7 @@ public override string GetSuggestedDestination(ILibrary library)
         {
             if (library != null && library is UnpkgLibrary unpkgLibrary)
```

Comment:
>library != null && l [](start = 16, length = 20)

Nit: is this part needed?

Reply:
Hm, I guess not, I can clean these up.  They've just been there since forever.